src/components/TodoItem/TodoStatus/TodoStatus.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+interface Props {
+  checked: boolean;
+  onChangeStatus: () => void;
```

Comment:
The onChangeStatus prop should have a more specific type, such as (checked: boolean) => void.